RF24Network.h

```diff
@@ -122,6 +122,11 @@
 // NO ACK Response Types
 //#define NETWORK_ACK_REQUEST 192
 
+/**
+ * Messages of this type indicate the network is being overrun with data & the RX FIFO has been flushed.
+ **/
+#define NETWORK_OVERRUN 160
```

Comment:
Here's my thoughts on the actual number. Beware, I still thinking in binary from exposing the STATUS byte...

The msg type is an 8-bit var. I was thinking we'd assert the top 3 bits (`0b11100000 = 0xE0 = 224`) and use the lower 5 bits to indicate the layer.

It doesn't really matter though. As long as the number isn't already used.

---
If you really want to get _fancy_ about it, we could declare an enum of `ErrorKind`s. That way the source code (and docs) would be a bit more organized about OSI errors.

This idea can wait I guess, until we flesh out what errors we want to propagate up the stack...

Reply:
> I was thinking we'd assert the top 3 bits (0b11100000 = 0xE0 = 224) and use the lower 5 bits to indicate the layer.

I think I'd need to see how this works out before forming an opinion. Knowing you it will all make sense in the end. :p 

> If you really want to get fancy about it, we could declare an enum of ErrorKinds.

That I understand, and its sounds like a good idea, but maybe for another PR.